ash/src/lib.rs

```diff
@@ -45,6 +50,11 @@
 //! * **debug** (default): Whether Vulkan structs should implement `Debug`.
 //! * **loaded** (default): Support searching for the Vulkan loader manually at runtime.
 //! * **linked**: Link the Vulkan loader at compile time.
+//! * **std** (default): Whether ash depends on the standard library (otherwise alloc is required)
+
+#[cfg_attr(feature = "std", allow(unused_extern_crates))]
+#[macro_use]
```

Comment:
Is `#[macro_use]` actually required here?

Reply:
That makes `vec![]` available without qualified `alloc::vec![]` references or imports.

How about adding `use alloc::vec` across the code-base? There's only one non-`std` use of the macro anyway.

Reply:
Grep tells me there's exactly three callsites for that, so `use`ing/qualifying seems fine.